cmd/cmd.go

```diff
@@ -92,13 +94,86 @@ var newAddress = &cobra.Command{
 	},
 }
 
-
 var burn = &cobra.Command{
-	Use:"burn",
-	Short:"Burns the specied amount of FCT into PNT",
-	Args: CombineCobraArgs(CustomArgOrderValidationBuilder(true, ArgValidatorFCTAddress, ArgValidatorFCTAmount)),
+	Use:   "burn <fct address> <fct amount>",
+	Short: "Burns the specied amount of FCT into PNT",
+	Long: "Burning FCT will turn it into PNT. The PNT burn address is an EC address, and the transaction has " +
+		"an input with # of FCT, and an output of 0 EC. This means the entire tx input becomes the fee. " +
+		"This command costs FCT, so be careful when using it.",
+	Example: "pegnet burn FA3EPZYqodgyEGXNMbiZKE5TS2x2J9wF8J9MvPZb52iGR78xMgCb 1",
+	// TODO: Verify this functionality.
+	ValidArgs: ValidOwnedFCTAddresses(),
```

Comment:
What does this do? Are you not exactly checking that in the `Run` function?

Reply:
It's for the auto complete. I'm still trying to to determine the best method to setup autocomplete, as currently you have to run the auto complete installation with walletd up. Then you can tab fill your addresses

Reply:
It doesn't actually prevent you from using another argument by the way. It just provides a list of valid ones on <tab>

Reply:
Ok got it, thanks.